ground/gcs/src/plugins/boards_brainfpv/brainre1.cpp

```diff
@@ -77,6 +77,22 @@ bool BrainRE1::queryCapabilities(BoardCapabilities capability)
     case BOARD_CAPABILITIES_OSD:
     case BOARD_CAPABILITIES_UPGRADEABLE:
         return true;
+    case BOARD_CAPABILITIES_MAGS:
+        {
+            ExtensionSystem::PluginManager *pm = ExtensionSystem::PluginManager::instance();
+            UAVObjectManager *uavoManager = pm->getObject<UAVObjectManager>();
+            HwBrainRE1 *hwBrainRE1 = HwBrainRE1::GetInstance(uavoManager);
+            Q_ASSERT(hwBrainRE1);
```

Comment:
Would prefer:
```c
if (!hwBrainRE1) {
    qWarning() << "Failed to get HwBrainRE1";
    Q_ASSERT(false);
    return false;
}
```
The qWarning notifies the cloud service (via UsageStats via DebugGadget) so we can track problems users are encountering without requiring them to actively feedback. It also prints to the console; handy if you don't have a debugger connected when you hit the Q_ASSERT (in a debug build). Finally, the return will be hit in release builds (Q_ASSERT is empty statement), which you did already handle on the next line.